Octo-Bot/packet-generator-bot/bot/helper/ReplayHelper.py

```diff
@@ -0,0 +1,9 @@
+from scapy.utils import rdpcap
+from os import path
+
+
```

Comment:
Can you a little documentation for this function?
Including expected inputs/arguments.

Reply:
Added!